projects/demo/src/modules/components/input-tag/examples/4/index.ts

```diff
@@ -42,11 +42,11 @@ export class TuiInputTagExample4 {
     readonly control = new FormControl([], createControlValidator(tagValidator));
 
     get filtered(): readonly string[] {
-        return this.filter(this.search, this.control.value);
+        return this.filterBy(this.search, this.control.value);
     }
 
     @tuiPure
-    private filter(search: string, value: readonly string[]): readonly string[] {
```

Comment:
oh, this lint rule has decided that it is array with `.filter()` function ?)

Reply:
yes 😅